### ISSUE TITLE
Time lag after invite sending before next invite is sent to next person

### ISSUE THREAD

**Issue by katedarracott** [charlielidbury/tempus-teething#18]

**Comment by charlielidbury** [comment]
clarification: make invites expire after a set time, allowing other people to take their place

**Comment by katedarracott** [comment]
Yes, that’s right. But it needs to show visisually who’s been invited, or does it just go round again?

Ps: no email footer on mobile. ;/)

**Comment by charlielidbury** [comment]
Yeah expiring would mean declining, so you'd be able to see them as having been invited already

**Comment by katedarracott** [comment]
Cool. Maybe set it to three days? With a ‘your invite will expire in 24 hrs’ notice after 2 days? Is that possible?

**Comment by charlielidbury** [comment]
Yeah that’s a good idea

Charlie
